api/app/course_responses.go

```diff
@@ -66,9 +66,11 @@ func (rs *CourseResource) newCourseListResponse(courses []model.Course) []render
 
 // SheetPointsResponse is response for performance on a specific exercise sheet
 type SheetPointsResponse struct {
-	AquiredPoints int `json:"acquired_points" example:"58"`
-	MaxPoints     int `json:"max_points" example:"90"`
-	SheetID       int `json:"sheet_id" example:"2"`
+	AquiredPoints int  `json:"acquired_points" example:"58"`
+	Graded        bool `json:"graded" example:"true"`
+	SheetGraded   int  `json:"sheet_graded" example:"42"`
```

Comment:
Should be 

```go
SheetGraded   bool  `json:"sheet_graded" bool:"false"`
```

Reply:
Boolean does not suffice, see my comment below.